packages/admin-ui/components/table/MeasureFieldTable.tsx

```diff
@@ -0,0 +1,48 @@
+import { useMemo } from 'react';
+import { Table, TableProps } from 'antd';
+import { MeasureFieldValue } from '@vulcan-sql/admin-ui/components/modals/AddMeasureFieldModal';
+
+type Props = Pick<TableProps<MeasureFieldValue>, 'dataSource'> &
+  Partial<Pick<TableProps<MeasureFieldValue>, 'columns'>>;
+
+export const getMeasureFieldTableColumns = () => {
+  return [
+    {
+      title: 'Name',
+      dataIndex: 'fieldName',
+    },
+    {
+      title: 'Expression',
+      dataIndex: 'expression',
+      render: (expression, record) => {
+        // TODO: clarify the interface with backend
+        const argumentFields = record.modelFields.map((field) => field.name);
+        const argumentTexts = argumentFields.join('.');
+        return `${expression}${argumentTexts ? `(${argumentTexts})` : ''}`;
+      },
+    },
+  ];
+};
+
+export default function MeasureFieldTable(props: Props) {
+  const { dataSource = [], columns = [] } = props;
+
+  const tableColumns = useMemo(
+    () => [...getMeasureFieldTableColumns(), ...columns],
+    [dataSource]
+  );
+
+  return (
+    <Table
+      rowKey={(record, index) => `${record.fieldName}-${index}`}
```

Comment:
For the current Ant.d V4 Table element, rowKey will no longer support **index**

Reply:
I checked the latest `rc-table`, it supports `index`, though Antd doesn't write it out in their documents. 🤔️
https://github.com/react-component/table/blob/master/src/interface.ts#L118

<img width="1337" alt="image" src="https://github.com/Canner/vulcan-sql/assets/9657305/665be9c8-34a9-4cf9-bd00-42c0fd4af38a">

Reply:
Hmm... I can see the `Warning: [antd: Table] `index` parameter of `rowKey` function is deprecated....` in the console.log. In fact, you will find that index is always 0.

Reply:
But this is not a big issue and does not affect the core functions. We can also improve it in subsequent PRs.

Reply:
I saw the `Warning` after restarting my process 😢, still don't know why Antd validating it.
I use map to give the key property before table component. [a96a4e0](https://github.com/Canner/vulcan-sql/pull/345/commits/a96a4e07b02643fb3246557afcfc07549f308b68)